barretenberg/cpp/src/barretenberg/client_ivc/client_ivc_auto_verify.test.cpp

```diff
@@ -62,17 +64,13 @@ TEST_F(ClientIVCAutoVerifyTests, Basic)
     ClientIVC ivc;
     ivc.auto_verify_mode = true;
 
-    {
-        // Initialize the IVC with an arbitrary circuit
-        Builder circuit_0 = create_mock_circuit(ivc);
-        ivc.accumulate(circuit_0);
-    }
+    // Initialize the IVC with an arbitrary circuit
+    Builder circuit_0 = create_mock_circuit(ivc, /*is_kernel=*/false);
```

Comment:
Couldn't you have `is_kernel = false` by default for mock circuits so that you only need a second parameter to this function when the circuit is acctually a kernel and you need to set it to `true`?

Reply:
I could but the method already has a 3rd argument with a default value and I try to avoid having methods with multiple default arguments when possible. This whole test suite will be deleted soon enough anyway